crates/runtime-request-context/src/context.rs

```diff
@@ -143,6 +144,19 @@ impl RequestContext {
         REQUEST_CONTEXT.scope(self, f).await
     }
 
+    /// Wraps provided stream with the current request context.
+    pub fn scope_stream<S>(self: Arc<Self>, stream: S) -> impl Stream<Item = S::Item>
+    where
+        S: Stream,
+    {
+        stream! {
```

Comment:
Rework this to not use the `stream!` macro.

Reply:
It breaks the use of rust-analyzer hints in the IDE, and generally makes it harder to debug issues that occur inside of the macro.